fpdf/drawing.py

```diff
@@ -0,0 +1,1023 @@
+from contextlib import contextmanager
+import enum
+import math
+
+
+def _clamp8(value):
```

Comment:
I guess the "8" in the name refers to 8bits?

The docstring would be something like: "Clamp a value inbetween 0 & 255 (excluded)"

Reply:
This is correct, it's to ensure we don't produce out-of-range color values. Thinking about it more, instead of clamping, it may be better to raise a `ValueError` for out-of-range values here.

Reply:
The color handling was reworked to do all of the range checking after conversion to floating point values, so this function doesn't exist any more. The range checking now raises a `ValueError` for out-of-range values instead of silently clamping them to the valid range.